src/Microsoft.Diagnostics.TestHelpers/TestHost/TestDump.cs

```diff
@@ -31,7 +31,7 @@ public TestDump(TestConfiguration config)
             _serviceContainer.AddService<ISymbolService>(_symbolService);
 
             // Register all the services and commands in the Microsoft.Diagnostics.DebugServices.Implementation assembly
-            _serviceManager.RegisterExportedServices(typeof(Target).Assembly);
+            _serviceManager.RegisterAssembly(typeof(Target).Assembly);
```

Comment:
It can do more than just Register export services.  All the hosts register a NotifyExtensionLoad event that calls the CommandService to add all the commands in the assembly. I like the name shorter also.